org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -534,18 +534,19 @@ SsoSynapseDevAthenaUser:
   Parameters:
     instanceArn: !Ref instanceArn
     principalId: !Ref synapseDevAthenaGroup
-    permissionSetName: 'athena-user-dev'
+    permissionSetName: 'synapse-dw-athena-dev'
+    permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-synapse-dw-athena-user-permission-set-arn' ]
```

Comment:
remove this, if you are creating a permission set (by defining the `permissionSetName` parameter) then you don't use an existing `permissionSetArn`

Reply:
> remove this, if you are creating a permission set (by defining the `permissionSetName` parameter) then you don't use an existing `permissionSetArn`

Since I want `permissionSetArn` I instead removed `permissionSetName`.

Reply:
The `!CopyValue` function copies an existing cloudformation output value which means that the cloudformation output must exist to use that function.  In the previous version of this PR it did exist with the creation of the shared `SsoSynapseDWAthenaUser` resource however since you removed it in this version of the PR it no longer exists.

You have two options to resolve this..
1. Use `permissionSetName` instead of `permissionSetArn`
2. Restore `SsoSynapseDWAthenaUser` then you can use `permissionSetArn`, also remove `permissionSetName`

Reply:
Thanks for the feedback.  I read through the script, [aws-sso.njk](https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/SSO/aws-sso.njk), and am not sure what the right thing is to pass in:  It looks like `aws-sso.njk` allows passing in a permission set, an AWS managed policy, or an in-line policy.  What I would like to pass in is a user-managed policy, `synapse-athena-user-access-policy`, defined [here](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/600-access/_tasks.yaml#L221). Can I pass `aws-sso.njk` a reference to this policy?  Or do I have to define a permission set that 'wraps' the policy and pass in the permission set ARN?  Or something else?